website/docs/02-separation-logic-and-biabduction.md

```diff
@@ -0,0 +1,271 @@
+---
+id: separation-logic-and-bi-abduction
+title: Separation logic and bi-abduction
+---
+
+export const Math = ({ code }) => (
+<img
+src={`https://math.now.sh?from=${encodeURIComponent(code)}&color=mediumslateblue`}
+style={{ height: '100%', verticalAlign: "middle" }}
+/>
+);
+
+- [Separation logic](separation-logic-and-bi-abduction#separation-logic)
+- [Bi-abduction](separation-logic-and-bi-abduction#bi-abduction)
+- [Technical papers](separation-logic-and-bi-abduction#technical-papers)
+
+## Separation logic
+
+Separation logic is a novel kind of mathematical logic which facilitates reasoning about
+mutations to computer memory. It enables scalability by breaking reasoning into chunks
+corresponding to local operations on memory, and then composing
+the reasoning chunks together.
+
+Separation logic is based on a logical connective <Math code="\\( * \\)" /> called the _separating conjunction_ and pronounced "and separately". Separation logic formulae are interpreted over program allocated heaps. The logical formula
+<Math code="\\( A*B \\)" /> holds of a piece of program heap (a heaplet) when it can be divided into two sub-heaplets described by <Math code="\\(A\\)" /> and <Math code="\\(B\\)" />.
```

Comment:
All the `\\(` and `\\)` in this file should be deleted: they show up as parentheses in the rendered html, but originally they were only indicating to switch to LaTeX mode between these markers and not meant to be rendered. With your changes we have `<Math code="...">` instead so `\\(` and `\\)` should be deleted.

Reply:
Fixed